### PR TITLE
RavenDB-21189 - RachisTests.AddNodeToClusterTests.RemoveNodeWithDb

### PR DESCRIPTION
**Pull request by lastav5**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-21189

### Additional description

Continuation of https://github.com/ravendb/ravendb/pull/17312
Meant to be a specific v6.0 fix on top of the 5.4 PR.

### Type of change

- Bug fix

### How risky is the change?

- Low 

### Backward compatibility

- Non breaking change

### Is it platform specific issue?

- No

### Documentation update

- No documentation update is needed

### CODE REVIEW

**Comment by ppekrol** [comment]
Is this a problem in 5.4 as well?

File: src/Raven.Client/Http/Topology.cs [hunk 1]
```diff
@@ -7,5 +7,10 @@ public sealed class Topology
         public long Etag;
         public List<ServerNode> Nodes;
         public List<ServerNode> Promotables = new List<ServerNode>();
+
+        public override string ToString()
+        {
+            return $"{{{nameof(Nodes)}: [{string.Join(",", Nodes)}], {nameof(Promotables)}: [{string.Join(",", Promotables)}], {nameof(Etag)}: {Etag}}}";
```
Comment (karmeli87) [inline src/Raven.Client/Http/Topology.cs]: I know it is debug.. but still, can `Nodes` be `null` here?

**Review by karmeli87 — APPROVED** [review]
Should target 5.4

**Comment by lastav5** [comment]
> Is this a problem in 5.4 as well?

Apparently it is but because the topology cache in 5.4 is by default not disabled it saves this function from throwing by returning here: https://github.com/ravendb/ravendb/blob/v5.4/src/Raven.Client/Http/RequestExecutor.cs#L832

I'll target 5.4 after dealing with failing tests

File: src/Raven.Client/Http/Topology.cs (hunk 1 above)
Comment (lastav5) [reply src/Raven.Client/Http/Topology.cs]: Fixed

**Comment by karmeli87** [comment]
This will be part of the merge here https://github.com/ravendb/ravendb/pull/17353

**Pull request closed without merging**